pan-domain-auth-play/src/main/scala/com/gu/pandomainauth/service/googleAuthModel.scala

```diff
@@ -26,7 +26,9 @@ object Token {
     (JsPath \ "id_token").read[String]
   )(Token.apply _)
 
-  def fromJson(json:JsValue):Token = Json.fromJson[Token](json).get
+  def fromJson(json:JsValue):Token = {
```

Comment:
Don't think anything really changes here does it? Can probably leave that change out.